### ISSUE TITLE
train_dataset_subjects_and_objects

### ISSUE THREAD

**Issue by zahramahani** [davidgolub/SimpleQA#8]
hi, can I ask you to put subjects_names, objects_names, and all entities for the train and validation set too?
